tox.ini

```diff
@@ -0,0 +1,27 @@
+[flake8]
+max-line-length = 88
+
+[tox]
+skipsdist = True
+envlist = lint,functional
+
+[testenv]
+setenv =
+    PYTHONPATH={toxinidir}:{toxinidir}/lib
+    PYTHONBREAKPOINT=ipdb.set_trace
+
+[testenv:lint]
+deps =
+    black
+    flake8
+commands =
+    flake8 {toxinidir}/lib {toxinidir}/tests
+#    black --check {toxinidir}/lib {toxinidir}/tests
```

Comment:
Did you leave `black` commented out so it didn't muddy the meat of this PR?

Reply:
Indeed I did.  Black reformat applied and `tox.ini` updated.